lib/service/AthleteApi.dart

```diff
@@ -4,6 +4,9 @@ import 'dart:convert';
 import 'package:http/http.dart' as http;
 
 class AthleteApi {
+  static final String _baseURL = 'https://db.athletex.io';
+  // static final String _BaseURL = 'http://139.99.74.201:8080';
```

Comment:
Just delete this comment it is not needed anymore

Reply:
Completed